pkg/fatfs/fatfs_vfs/fatfs_vfs.c

```diff
@@ -45,13 +56,12 @@ static int _mount(vfs_mount_t *mountp)
 
     fatfs_desc_t *fs_desc = (fatfs_desc_t *)mountp->private_data;
 
-    char volume_str[FATFS_MAX_VOL_STR_LEN];
-    snprintf(volume_str, sizeof(volume_str), "%d:/", fs_desc->vol_idx);
+    _build_abs_path(fs_desc, "");
```

Comment:
In this one, you replaced the `volume_str` by using `fs->abs_path_str_buff` is it ok to use this one ? Just asking I do not know whet it should be used for.

Reply:
`fs->abs_path_str_buff` is used as a string buffer for building the absolute paths for the library calls in other parts of the VFS wrapper. Since it is already allocated inside the struct, I didn't see any point in adding another string buffer on the stack, even if it is only a few bytes long.

Reply:
Ok, from the type definition, it looks like it should be used for this anyway.